src/tests/ordered-controller-params/valid-cases/valid-only-cases/index.ts

```diff
@@ -0,0 +1,5 @@
+import path from "path";
```

Comment:
Bi li ovaj folder triba bit susjedan `valid-cases` folderu?

Reply:
Ne vidim zasad nijedan pretjerano dobar razlog da bude unutar `valid-cases`, ali isto tako ni da bude izvan njega.
Stavio sam ga unutar čisto jer je to vrsta validnih slučajeva pa semantički ima smisla da je unutar njega.

Reply:
Ok